src/test/ui/associated-type-bounds/inside-adt.rs

```diff
@@ -0,0 +1,33 @@
+// compile-fail
```

Comment:
I think it's better to have the actual errors per line, as it's more specific, rather than just a catch all. `error-pattern` should only be for errors that aren't ones you explicitly want to test.

Reply:
Yeah that makes sense. :)